x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/EsqlFeatures.java

```diff
@@ -68,7 +68,7 @@ public class EsqlFeatures implements FeatureSpecification {
     /**
      * Support for spatial aggregation {@code ST_CENTROID}. Done in #104269.
      */
-    private static final NodeFeature ST_CENTROID = new NodeFeature("esql.st_centroid");
+    private static final NodeFeature ST_CENTROID_AGG = new NodeFeature("esql.st_centroid_agg");
```

Comment:
Good thing we marked that as experimental in the docs and can just make breaking changes!

Reply:
Personally I hope in future to allow the same name to work as both scalar and aggregating functions, but by then we'll need to go through a deprecation phase to remove the _AGG version. Renaming now gives us more options going forward, with less bwc issues.